### PR TITLE
feat: added mission_command route

### PR DESCRIPTION
**Pull request by timigod**
## Description
- Added a new route that will accept a `user id`, a `mission id`, and a command.

- If `user_id` doesn't match the mission's user_id, it returns a 401.

- If the `command` string is `takeoff_pickup` and the current status of the vehicle is `waiting_pickup`, it updates the mission and vehicle status to `takeoff_pickup`.

- Also fixed `missionProgress.js`. The fix stops the mission/vehicle status from moving from `waiting_pickup` to `travelling_droppoff` automatically without any user input/command.

### CODE REVIEW

File: server/controllers/MissionController.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const { createMissionUpdate } = require('../store/mission_updates');
+const { getVehicle, updateVehicleStatus } = require('../store/vehicles');
+
+
+const command = async (req, res) => {
+  const { user_id, mission_id, command} = req.query;
+  const mission = await getMission(mission_id);
+  const vehicle = await getVehicle(mission.vehicle_id);
+
+  if (command === 'takeoff_pickup' && vehicle.status === 'waiting_pickup'){
```
Comment (TalAter) [inline server/controllers/MissionController.js]: Add a test that user_id matches the mission's user_id

Comment (timigod) [reply server/controllers/MissionController.js]: I did this and somehow didn't push it. Just did.

File: server/controllers/MissionController.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const { createMissionUpdate } = require('../store/mission_updates');
+const { getVehicle, updateVehicleStatus } = require('../store/vehicles');
+
+
+const command = async (req, res) => {
+  const { user_id, mission_id, command} = req.query;
+  const mission = await getMission(mission_id);
+  const vehicle = await getVehicle(mission.vehicle_id);
+
+  if (user_id !== mission.user_id) res.sendStatus(401);
```
Comment (TalAter) [inline server/controllers/MissionController.js]: Does `res.sendStatus()` also exit/returns? If not, we should `return` out of this function so it doesn't continue to do the mission/vehicle updates even when user is unauthorized.

Comment (timigod) [reply server/controllers/MissionController.js]: It doesn't exit/return. Pushing that now.

**Pull request merged**